app/src/main/java/org/thoughtcrime/securesms/components/ProfilePictureView.kt

```diff
@@ -53,19 +53,19 @@ class ProfilePictureView @JvmOverloads constructor(
 
     fun update(recipient: Recipient) {
         this.recipient = recipient
-        recipient.run { update(address, isClosedGroupRecipient, isOpenGroupInboxRecipient) }
+        recipient.run { update(address, isLegacyGroupRecipient, isCommunityInboxRecipient) }
```

Comment:
Is this right? This means only legacy groups will set this flag. Shouldn't GroupsV2 be considered groups for this profile logic? Or do we need a separate flag for v@ to be handled differently later in the update logic?